### PR TITLE
fix: remove email request prompt

### PR DESCRIPTION
**Pull request by pbosio**
#641 
remove email request prompt hud since we are already asking for user's email during sign-up

### CODE REVIEW

File: kernel/packages/shared/types.ts [hunk 1]
```diff
@@ -440,14 +440,13 @@ export enum HUDElementID {
   CONTROLS_HUD = 18,
   EXPLORE_HUD = 19,
   HELP_AND_SUPPORT_HUD = 20,
-  EMAIL_PROMPT = 21,
```
Comment (menduz) [inline kernel/packages/shared/types.ts:454]: Please leave this untouched, we should decouple the kernel change from the renderer change, to do so, we always should make enums append only

**Review by menduz — CHANGES_REQUESTED** [review]
Looking good, please restore the enum and add a "// @deprecated" to the declaration

File: kernel/packages/shared/types.ts (hunk 1 above)
Comment (pbosio) [reply kernel/packages/shared/types.ts:454]: OKs! Good to know! I'll do that.
I think that probably it's not a very good idea to keep using this Enum then
Cause I still need to change the value for the following elements to match unity's

Comment (menduz) [reply kernel/packages/shared/types.ts:454]: Is it already merged to master in unity renderer? I rather roll it back, it breaks compatibility with .zone

**Review by menduz — APPROVED** [review]
Awesome, thank you 🙌

**Pull request merged**